client/my-sites/checklist/onboardingChecklist.js

```diff
@@ -31,6 +31,7 @@ const tasks = {
 		url: '/me',
 		image: '/calypso/images/stats/tasks/upload-profile-picture.svg',
 		tour: 'checklistUserAvatar',
+		buttonText: 'Add Profile',
```

Comment:
Each of these texts are what we translate, ideally. Including punctuation if necessary.

In order to do that, we'll need to add the import in the `External dependencies` block:

```js
import { translate } from 'i18n-calypso';
```

Then wrap the texts as so: `translate( 'Add Profile' )`.